cross-chain/arbitrum/package.json

```diff
@@ -30,7 +30,9 @@
     "lint:config": "prettier --check '**/*.@(json|yaml)'",
     "lint:config:fix": "prettier --write '**/*.@(json|yaml)'",
     "prepack": "tsc -p tsconfig.export.json && hardhat export-artifacts export/artifacts",
-    "prepublishOnly": "echo TODO: Copy deployment artifacts to artifacts directory",
+    "export-artifacts:goerli": "yarn hardhat export-deployment-artifacts --network arbitrumGoerli",
```

Comment:
export-artifacts:goerli -> export-artifacts:arbitrumGoerli?

same for mainnet

Reply:
The reason for defining the name with `goerli` and `mainnet` is that it will combine both l1 and l2. The `export-artifacts:goerli` script should export deployment artifacts for both networks: `goerli` to `artifacts/l1` and `arbitrumGoerli` to `artifacts/l2`, to do that the script will be defined as follow:
```
"export-artifacts:goerli": "for i in goerli arbitrumGoerli; do yarn hardhat export-deployment-artifacts --network $i; done",
```

It is also a preparation for the automated deployment, where we have just `goerli` environment passed around (https://github.com/keep-network/tbtc-v2/blob/4523b83f2b4612d93d91ab55e254bb13cec632a1/.github/workflows/contracts.yml#L29-L31)

Reply:
ah, I see now, thx for explaining.